### DISCUSSION TITLE
[@types/applepayjs] Add billingContact to ApplePayPaymentMethodSelectedEvent

### DISCUSSION THREAD

**Discussion by vsashyn** [DefinitelyTyped/DefinitelyTyped#57506] [Issues with a @types package]
According to the [docs](https://developer.apple.com/documentation/apple_pay_on_the_web/applepaypaymentmethod#see-also) and results of local testing,  `ApplePayPaymentMethodSelectedEvent` has `paymentMethod` param with `billingContact` field, which is optional. `paymentMethod` will include `billingContact` if `postalAddress` is not required in `requiredShippingContactFields`.

My suggestion is to add `billingContact` to  [ApplePayPaymentMethod](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/applepayjs/index.d.ts#L438)

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
PR is ready #57517

**Reply by vsashyn** [reply]
@martin-badin cool, I've also prepared a PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57507 . It seems like we did it simultaneously 🙂 
@martin-badin just one small note regarding your changes, `billingContact` should be optional in `paymentMethod` because [it might be hidden by Apple Pay for privacy reasons](https://developer.apple.com/documentation/apple_pay_on_the_web/applepaypaymentmethodselectedevent/1778025-paymentmethod).
